com.unity.render-pipelines.high-definition/Documentation~/Compositor-User-Guide.md

```diff
@@ -38,18 +45,47 @@ To add a Sub-layer to a Composition Layer, select the Composition Layer and clic
 
 To remove a Sub-layer, select the Sub-layer and click the **Delete** button.<br/>**Note**: You can only delete Sub-layers this way and not Composition Layers. Instead, to delete a Composition Layer, remove the corresponding Texture2D property from the Composition Graph. 
 
+## Post-Processing and Sub-Layers
+Each one of the Sub-layers in a Composition Layer can have a unique set of post-processing effects. To configure this type of per-layer post-processing, you have to:
+- Add a unique Volume in your scene for every Sub-Layer that requires unique post-processing effects.
+- For each one of the Volumes, create and set a unique layer mask in the Unity Editor.
+- In the properties of each Sub-Layer, override and set the appropriate volume mask. 
+
+When post-processing is executed for a Sub-layer, only pixels with an alpha value that is greater than zero will be processed and will be affected by the post-processing operations. By selecting the "Clear Alpha" option in the Sub-layer Properties, you can ensure that post-processing will affect only the pixels that were drawn from this Sub-layer. Alternatively, if the "Clear Alpha" option is not selected, then the post-processing will also affect the Sub-layers that were previously drawn.
+
+Note that enabling expensive post-processing effects (such as Depth of Field and Bloom) in multiple Sub-layes will negatively impact the run-time performance. Therefore it is always more preferable to run the post-processing in groups of layers when possible, by smartly utilizing the "Clear Alpha" option in the Sub-layer Properties.
```

Comment:
Note that enabling resource-intensive post-processing effects (such as Depth of Field and Bloom) in multiple Sub-layers negatively impacts run-time performance. Therefore it is best practice to run post-processing for groups of layers when possible. To do this, use the **Clear Alpha** option in the Sub-layer Properties.